data_structures/linked_list/detect_n_removeloop.py

```diff
@@ -0,0 +1,98 @@
+# Python program to detect and remove loop in linked list 
+  
+# Node class  
+class Node: 
+  
+    # Constructor to initialize the node object 
+    def __init__(self, data): 
+        self.data = data 
+        self.next = None
+  
+class LinkedList: 
+  
```

Comment:
possibly add a docstring describing about how to use and what does it do, comments are not able to show what is the soul purpose. docstring might help